### PR TITLE
[OF-1074] Add fields required to create survey as a wizard. Add icons to questions

### PR DESCRIPTION
**Pull request by dmiskiew**
OF-1074

### CODE REVIEW

File: app/models/survey/answer.rb [hunk 1]
```diff
@@ -21,6 +23,9 @@ class Survey::Answer < ActiveRecord::Base
   before_save :check_single_choice_with_field_case
 
   def value
+    # TODO: number type option should calculate the value based on custom formula
```
Comment (SemionPar) [inline app/models/survey/answer.rb:26]: is this related to https://offcents.atlassian.net/browse/OF-1198?

File: spec/dummy/config/database.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# SQLite version 3.x
+#   gem install sqlite3
```
Comment (SemionPar) [inline spec/dummy/config/database.yml:2]: I wonder do we need all of this here, like rails server config, assets etc. if we are only using this as library

File: spec/services/survey/attempt_manager_spec.rb [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+  let!(:section_two) { survey.sections.where(position: 2).first }
+
+  let!(:questions_section_one) { section_one.questions }
+  let!(:questions_section_two) { section_two.questions }
+
+  let(:all_questions) { questions_section_one + questions_section_two }
+  let(:current_question) { nil }
+  let(:current_section) { nil }
+
+  describe '#create_by_survey_identifier' do
```
Comment (SemionPar) [inline spec/services/survey/attempt_manager_spec.rb:48]: Great that we have this tested! Do you think its worth the extra effort to hook this up to circleci? I think a simple ruby image with bundle install && rake test should suffice

File: spec/dummy/config/database.yml (hunk 2 above)
Comment (SemionPar) [reply spec/dummy/config/database.yml:2]: or is it actually used in offcents-api by active admin? I mean anything past Survey service

**Review by SemionPar — CHANGES_REQUESTED** [review]

File: app/models/survey/answer.rb (hunk 1 above)
Comment (dmiskiew) [reply app/models/survey/answer.rb:26]: Yes, exactly!

File: spec/dummy/config/database.yml (hunk 2 above)
Comment (dmiskiew) [reply spec/dummy/config/database.yml:2]: https://guides.rubyonrails.org/engines.html#testing-an-engine

File: spec/services/survey/attempt_manager_spec.rb (hunk 3 above)
Comment (dmiskiew) [reply spec/services/survey/attempt_manager_spec.rb:48]: Very good idea! Let me create a story for that

Comment (dmiskiew) [reply spec/services/survey/attempt_manager_spec.rb:48]: Done: https://offcents.atlassian.net/browse/OF-1249

File: spec/dummy/config/database.yml (hunk 2 above)
Comment (SemionPar) [reply spec/dummy/config/database.yml:2]: 👍

**Review by SemionPar — APPROVED** [review]

**Pull request merged**